userspace/libpman/include/libpman.h

```diff
@@ -241,6 +241,14 @@ extern "C"
 	 */
 	int pman_detach_signal_deliver(void);
 
+	/**
+	 * @brief Mark programs for uninteresting syscalls to no autoload
+	 *
+	 * @param ppm_sc_of_interest list of interesting ppm_sc.
```

Comment:
[nit] it's not that clear from this description how this list is used, and this comment (and others below) could be updated to reflect that it must contain true/false values for every ppm_sc 

It might also be worth reflecting this in the function signature

```
int pman_set_autoload_programs(const bool ppm_sc_of_interest[PPM_SC_MAX]);
```

Reply:
The original signature was gonna be the one you suggested, but `PPM_SC_MAX` is defined in scap.h and I'm not sure if including that here will cause any sort of conflicts, I'll try again.

I will also update the comments as suggested, I think the whole `ppm_sc_of_interest` is quite vaguely documented all over the code base, but that's no excuse for me slacking off.